economy/static/economy/css/economy_home.css

```diff
@@ -0,0 +1,89 @@
+.card {
+  display: flex;
+  align-items: center;
+
+  color: #697a92;
+
+  border: 2px solid #e3e7ef;
+  border-radius: 0.75px;
+  box-shadow: 0 3px 6px rgba(0, 0, 0, 0.16);
+
+  padding: 1rem;
+
+  margin: 0 0 1rem 0;
+}
+
+.card > * {
+  padding: 1rem;
+}
+
+.justify-center {
+  justify-content: center;
+}
+
+#economy-home-wrapper {
+  display: flex;
+  flex-direction: column;
+
+  align-items: center;
+}
+
+#economy-home-grid {
+  display: grid;
+  max-width: 80%;
+  grid-template-columns: 1fr 1fr;
+  gap: 10px;
+  grid-template-areas:
+    "tl tr"
+    "bl br";
```

Comment:
Preferably name the grid-template-areas with the content of the areas, instead of abbrreviations for top-left, right and bottom-left, right.